store/src/java/com/zimbra/util/ExternalVolumeInfoHandler.java

```diff
@@ -146,7 +204,9 @@ public void addServerProperties(VolumeInfo volInfo) throws ServiceException, JSO
             currentJsonObject.put("server/stores", currentJsonArray);
 
             // step 6: Set ldap attribute
+            writeServerJSONFile(provisioning.getLocalServer().getServerExternalStoreConfig(), DIR_PATH, PREV_FILE_PATH);
             provisioning.getLocalServer().setServerExternalStoreConfig(currentJsonObject.toString());
+            writeServerJSONFile(provisioning.getLocalServer().getServerExternalStoreConfig(), DIR_PATH, CURR_FILE_PATH);
```

Comment:
We can write currentJsonObject directly instead of getting data from provisioning.getLocalServer().getServerExternalStoreConfig()

Reply:
ok. Fixed